bash/scripts/publish_helm_chart.sh

```diff
@@ -79,16 +90,27 @@ update-chart() {
 
       component_chart_version="${latest_tag}"
       component_chart_repo="${component_chart}"
-      if [ "${chart_version}" != "${git_tag}" ]; then
-        # chart version has build data; is -dev variant
+      # if COMPONENT_REPO matches this component repo and COMPONENT_COMMIT is non-empty/non-null,
+      # this signifies we need to set component chart version to correlate with PR artifact
+      # shellcheck disable=SC2153
+      if [ "${COMPONENT_REPO}" == "${component_repo}" ] && [ -n "${COMPONENT_COMMIT}" ]; then
+        incremented_patch_version="$(( ${latest_tag: -1} +1))"
+        component_chart_version="${latest_tag%?}${incremented_patch_version}-sha.${COMPONENT_COMMIT:0:7}"
+        component_chart_repo="${component_chart}-pr"
+      elif [ "${chart_version}" != "${git_tag}" ]; then
+        # workflow chart version has build data; is -dev variant. assign component version/repo accordingly
         component_chart_version=">=${latest_tag}"
         component_chart_repo="${component_chart}-dev"
       fi
+
       perl -i -0pe 's/<'"${component_chart}"'-tag>/"'"${component_chart_version}"'"/g' "${chart}"/requirements.yaml
       perl -i -0pe 's='"${DEIS_CHARTS_BASE_URL}/${component_chart}\n"'='"${DEIS_CHARTS_BASE_URL}/${component_chart_repo}\n"'=g' "${chart}"/requirements.yaml
       helm repo add "${component_chart_repo}" "${DEIS_CHARTS_BASE_URL}/${component_chart_repo}"
     done
 
+    # DEBUG
+    cat "${chart}"/requirements.yaml
```

Comment:
@bacongobbler alas, this continues to help us investigate/troubleshoot failed chart packaging... I think we want to keep this `cat` for the time being.  Therefore, I'll remove the `# DEBUG` comment above.